<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UI1v1</class>
 <widget class="QWidget" name="UI1v1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>935</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UI1v1</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="live1v1_widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_17">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="whiteboard1v1_widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="full1v1_widget" native="true">
           <property name="styleSheet">
            <string notr="true">border-image: url(:/LoginWindow/images/videoback.png);</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_19">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>325</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QWidget" name="fullS1v1_widget" native="true">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_21">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>326</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>1</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>1</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(5, 157, 213);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="livetool1v1_widget" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>60</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_20" stretch="0,0,0,0,0,0">
            <property name="spacing">
             <number>30</number>
            </property>
            <property name="leftMargin">
             <number>9</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>9</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="spacing">
               <number>10</number>
              </property>
              <item>
               <widget class="QLabel" name="label_NetTitle">
                <property name="text">
                 <string>网络状态:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_NetState">
                <property name="text">
                 <string>正常</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_27">
              <item>
               <widget class="QCheckBox" name="AudioOut_checkBox">
                <property name="minimumSize">
                 <size>
                  <width>29</width>
                  <height>19</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>29</width>
                  <height>19</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>扬声器</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="AudioOut_pushButton">
                <property name="minimumSize">
                 <size>
                  <width>18</width>
                  <height>19</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>18</width>
                  <height>19</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>扬声器参数</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_25">
              <item>
               <widget class="QCheckBox" name="Audio1v1_checkBox">
                <property name="minimumSize">
                 <size>
                  <width>24</width>
                  <height>19</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>24</width>
                  <height>19</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>麦克风</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="Audio1v1_pushButton">
                <property name="minimumSize">
                 <size>
                  <width>18</width>
                  <height>19</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>18</width>
                  <height>19</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>麦克风参数</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_26">
              <item>
               <widget class="QCheckBox" name="video1v1_checkBox">
                <property name="minimumSize">
                 <size>
                  <width>29</width>
                  <height>19</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>29</width>
                  <height>19</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>摄像头</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="video1v1_pushButton">
                <property name="minimumSize">
                 <size>
                  <width>18</width>
                  <height>19</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>18</width>
                  <height>19</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>摄像头参数</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPushButton" name="btn_GetFrameState">
                <property name="minimumSize">
                 <size>
                  <width>29</width>
                  <height>19</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>29</width>
                  <height>19</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="line_label">
        <property name="minimumSize">
         <size>
          <width>1</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(5, 157, 213);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="chatcamera1v1_widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="camera1v1_widget" native="true">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>112</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>112</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="cameraT_widget" native="true">
              <property name="styleSheet">
               <string notr="true">border-image: url(:/LoginWindow/images/camera_1.png);</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_22">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="minimumSize">
               <size>
                <width>1</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>1</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(5, 157, 213);</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="cameraS_widget" native="true">
              <property name="styleSheet">
               <string notr="true">border-image: url(:/LoginWindow/images/camera_1.png);</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_23">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>1</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>1</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(5, 157, 213);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="chat1v1_widget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_18">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
